x-pack/plugins/remote_clusters/public/sections/components/remote_cluster_form/validators/validate_name.js

```diff
@@ -8,7 +8,7 @@ import React from 'react';
 import { FormattedMessage } from '@kbn/i18n/react';
 
 export function validateName(name) {
-  if (name == null || !name.trim()) {
+  if (!name || !name.trim()) {
```

Comment:
Don't ask me why I did this change. But I guess it doesn't do any harm 😊

Reply:
Actually, I do remember... I wanted to catch the case where `name` is not a string (undefined) and to avoid calling `if (name.includes` on L20.